src/index.js

```diff
@@ -1,7 +1,168 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import Map from './map.js'
+import leafletPip from  '@mapbox/leaflet-pip'
+import L from 'leaflet'
+import borderData from './border.js'
+
+const wholeThang = {
+  display: "flex",
+
+}
+const img = {
+  height : "500px"
+}
+
+
+class App extends React.Component {
+  state = { 
+    markerPosition: { lat: 44.5588,lng: -72.5778 },
+    mapCenter: {lat: 44.5588, lng: -72.5778},
+    town: {}  
+  };
+  
+  startGame = () => {
+    document.getElementById("start-button").disabled = true
+    document.getElementById("guess-button").disabled = false
+    document.getElementById("quit-button").disabled = false
+    let randoLat = this.getRandoLat()
+    let randoLng = this.getRandoLng()
+    let layerArray = leafletPip.pointInLayer ([randoLng, randoLat],L.geoJSON(borderData))
+    while (layerArray.length === 0){
+      randoLat = this.getRandoLat()
+      randoLng = this.getRandoLng()
+      layerArray = leafletPip.pointInLayer ([randoLng, randoLat],L.geoJSON(borderData))
+    }
+    this.setState({
+      mapCenter:{
+        lat: randoLat,
+        lng:randoLng
+      },
+      markerPosition:{
+        lat: randoLat,
+        lng:randoLng
+    }
+  })
+  }
```

Comment:
For readability leave a blank line between separate method definitions. It makes your code a little easier for our poor human brains to scan through.